src/objects/creatures/iceblock.js

```diff
@@ -16,3 +17,19 @@ IceBlock.prototype.stomp = function() {
     this.sprite.setRow(1);
     this.hSpeed = 0;
 }
+
+IceBlock.prototype.horizontalCollision = function(item) {
+    if (!(item instanceof Coin)) {
```

Comment:
coins are not the only elements without collision. But elements do have a collision property you can check instead